src/BoundaryConditions/flat_extrapolation_open_boundary_matching_scheme.jl

```diff
@@ -57,14 +57,17 @@ end
 
     return c + Δc
 end
-@inline spacing_factor(args...) = 1/2
-@inline spacing_factor(Δ::AbstractArray, ::Val{:right}) = @inbounds Δ[end] / (Δ[end-1] + Δ[end-2])
-@inline spacing_factor(Δ::AbstractArray, ::Val{:left})  = @inbounds Δ[1]   / (Δ[2]     + Δ[3]    )
+
+c = Center()
```

Comment:
```suggestion
const c = Center()
```

Reply:
I think this comment is on an outdated version